notify/impl.go

```diff
@@ -62,6 +66,79 @@ func Build(confs []*config.Receiver, tmpl *template.Template) map[string]Fanout
 
 const contentTypeJSON = "application/json"
 
+// End-users should not be exposed to Go's type system,
+// as this will confuse them and prevent simple things like
+// simple equality checks to fail. Map everything to float64/string.
+type templateData struct {
+	Status                      string
+	Alerts                      []templateAlert
+	AlertCommonLabels           map[string]string
+	AlertCommonLabelnamesSorted []string
```

Comment:
The `Sorted` suffix just seems noisy. That can be part of the documentation. As the user has no influence on iteration order anyway the info is not helping much anyway.